<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicateClass</class>
 <widget class="QWidget" name="CommunicateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Communicate</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QCheckBox" name="checkBox_sender">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>发送端</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>99</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>网卡：</string>
   </property>
  </widget>
  <widget class="QLabel" name="receiveLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_receive">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>105</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>接收端</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_IP">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="communicate.qrc"/>
 </resources>
 <connections/>
</ui>
